en/editor/publish/publish-xiaomi-quick-game.md

```diff
@@ -23,49 +21,49 @@ __Cocos Creator__ officially supports the release of games to the **Xiaomi Quick
+    Click on the **xiaomi-quick-game** below to expand the parameter configuration of **Xiaomi Quick Game**.
 
-  ![](./publish-xiaomi-quick-game/xiaomi_options.jpg)
+    ![](./publish-xiaomi-quick-game/xiaomi_options.jpg)
 
 The specific filling rules for the relevant parameter configuration are as follows:
 
 - **App Package Name**: the format of the **App Package Name** is `com.yourcompany.projectname`. This option is required and will be filled in according to the developer's needs.
 
-  > **Note**: starting from the platform version number **1062**, Xiaomi Quick Game needs to use the official **App Package Name**, otherwise the error of `Data loading exception, please click retry` will be reported during debugging. You can refer to the [Xiaomi Quick Game App Package Name Application](https://forum.cocos.org/t/topic/81887) documentation for details.
+    > **Note**: starting from the platform version number **1062**, Xiaomi Quick Game needs to use the official **App Package Name**, otherwise the error of `Data loading exception, please click retry` will be reported during debugging. You can refer to the [Xiaomi Quick Game App Package Name Application](https://forum.cocos.org/t/topic/81887) documentation for details.
 
 - **Desktop Icon**: is required. Click the **search icon** button at the back of the input box to select the icon you want. When building, the Desktop Icon will be built into the Xiaomi Quick Game project. It is suggested to use a `PNG` image for the **Desktop Icon**.
 
 - **App Version Name**: is required. **App Version Name** is the real version, such as: **1.0.0**.
 
 - **App Version Number**: is required. **App Version Number** is different from the **App Version Name**, and the **App Version Number** is mainly used to distinguish the version update. Each time when you submit audit, the **App Version Number** is at least 1 higher than the value of the last submitted audit. It must not be equal to or less than the value of the last submitted audit, and it is recommended that the **App Version Number** be recursively incremented by 1 each time when the audit is submitted.
 
-  > **Note**: the **App Version Number** must be a positive integer.
+    > **Note**: the **App Version Number** must be a positive integer.
 
 - **Supported Minimum Platform Version Number**: is required. According to the requirements of Xiaomi Quick Games, this value must be greater than or equal to **1050**.
 
 - **Small Packet Mode**: is optional. The in-package volume of the quick-game contains code and assets that cannot exceed 10M, and assets can be loaded via network requests. **Small Packet Mode** is to help developers keep the script files in the quick game package, other assets are uploaded to the remote server, and downloaded from the remote server as needed. And the download, cache and version management of remote assets, __Cocos Creator__ has already helped the developer. What the developer needs to do is the following steps:
```

Comment:
And the download, cache and version management of remote assets, __Cocos Creator__ has already helped the developer. What the developer needs to do is the following steps:
-->
__Cocos Creator__ has already helped the developer with the download, cache, and version management of remote assets. The developer needs to do is the following steps: